packages/backend/src/tenants/service.test.ts

```diff
@@ -34,20 +35,20 @@ describe('Tenant Service', (): void => {
       ...Config,
       dbSchema
     })
-    appContainer = await createTestApp(deps)
-    tenantService = await deps.use('tenantService')
     knex = await deps.use('knex')
     config = await deps.use('config')
+    appContainer = await createTestApp(deps)
+    tenantService = await deps.use('tenantService')
     authServiceClient = await deps.use('authServiceClient')
     tenantSettingsService = await deps.use('tenantSettingService')
   })
 
   afterEach(async (): Promise<void> => {
-    await truncateTables(knex, true, dbSchema)
+    await truncateTables(deps, { truncateTenants: true })
   })
 
   afterAll(async (): Promise<void> => {
-    nock.cleanAll()
+    nock.cleanAll() // TODO: rm?
```

Comment:
I think we can remove since it seems as though its not used here

Reply:
whoops, yes. meant to include that.